join.html

```diff
@@ -70,24 +91,105 @@ <h2 class="title">Curriculum</h2>
+                Teach LA has a vibrant developer team culture that we'd love to have you join! Our dev team has two key focuses:
+            </p>
+                <ol>
+                    <li>Create software tools for <b>accessible education</b></li>
+                    <li>Provide a <b>welcoming space</b> for developers to grow</li>
+                </ol>
+            <p>Here's what we do:
             </p>
             <ul>
                 <li>Maintain our larger projects: the Teach LA Editor and its backend</li>
-                <li>Create interactive learning modules to present complicated topics to kids</li>
-                <li>Propose, develop, and deliver software to help Teach LA's mission!</li>
-                <li>Trainings and mentorship from lead developers in a variety of technologies!</li>
-                <li>Flexible committment - as much as you want, when you want</li>
+                <li>Learning Labs: Create interactive learning modules to present complicated topics like machine learning and cybersecurity to kids</li>
+                <li>Maintain this website</li>
+                <li>Propose, develop, and deliver software to help Teach LA's mission</li>
+                <li>Trainings and mentorship from lead developers in a variety of technologies</li>
             </ul>
+            <p>
+                Our dev team members have flexible schedules - pick up as much work as you want, when you want. Here's how to get started:
+            </p>
+            <ol>
+                <li><a class="tla-link" href="{{site.baseurl}}/docs/dev-setup">Set up your dev environment</a></li>
+                <li><a class="tla-link" href="{{site.baseurl}}/docs/static-setup">Add yourself to our team page</a></li>
+                <li>Reach out to the <a class="tla-link" href="{{site.baseurl}}/team#krash">dev team director, Leo</a></li>
+            </ol>
             <a class="button flex-center" href="{{site.dev_team_interest}}"><span class="fa fa-file-alt"></span> Dev Team Interest Form</a>
         </div>
+        <div class="col">
+            <img class="img-responsive" src="{{ site.baseurl }}/img/undraw/pair-programming.svg" alt="Male and female programmers working together at computer in nice office with plants"/>
+        </div>
+    </div>
+
+    <h2 class="title">Events</h2>
+    <div class="row">
+        <div class="col">
+            <img class="img-responsive" src="{{site.baseurl}}/img/undraw/join-events.svg" alt="Man pointing to events on a calendar"/>
+        </div>
+        <div class="col no-gutter">
+            <p>
+                Our special events team hosts lots of <b>stand-alone</b> events and <b>short courses</b>, usually involving <b>collabs with other orgs</b>. We aim to <b>introduce CS</b> to kids and show them what it’s all about! <a class="tla-link" href="{{site.baseurl}}/citylab-cyber">Here</a> is an example of a past event that we’ve had. Interested? Here’s what we do: 
```

Comment:
A super small tidbit: I would avoid using inner content like `<a ...>Here</a>` since it's not very accessible for screenreaders! Instead `<a>Here is an example</a>` is already more descriptive, but I think something like `<a ...>Cyber Day at CityLab</a>` is also good!